layouts/shortcodes/python_version.html

```diff
@@ -0,0 +1 @@
+{{ .Site.Params.pythonVersion }}
```

Comment:
I'm not sure I see the point of this indirection?

Reply:
It's just a little syntactic sugar. Means you can use `{{% python_version %}}` where it's needed rather than `{{ .Site.Params.pythonVersion }}